batchlib/segmentation/torch_prediction.py

```diff
@@ -77,6 +91,9 @@ def run(self, input_files, output_files, gpu_id=None, batch_size=1, normalize=de
             model = self.load_model(device)
             model.to(device)
 
-        input_batches, output_batches = files_to_jobs()
-        for in_batch, out_batch in tqdm(zip(input_batches, output_batches), total=len(input_files)):
-            self.predict_images(in_batch, out_batch, model, normalize, device)
+            #input_batches, output_batches = files_to_jobs()
```

Comment:
here I don't quite got the semantics of `files_to_jobs` so just commented it out for mow

Reply:
Ah I guess I forgot to fill that in :D. I think what I wanted to do here does not quite work.
I will just merge and fix it myself, that's much easier.